x-pack/legacy/plugins/index_management/public/sections/home/templates_list/templates_list.tsx

```diff
@@ -20,9 +21,19 @@ import { TemplatesTable } from './templates_table';
 import { loadIndexTemplates } from '../../../services/api';
 import { Template } from '../../../../common/types';
 import { trackUiMetric, METRIC_TYPE } from '../../../services/track_ui_metric';
```

Comment:
Minor suggestion: you might find this module's interface to be simpler if it exports `trackClick`, `trackLoad`, etc.

Reply:
@cjcenizal Thanks for suggestion! I will update on another PR.